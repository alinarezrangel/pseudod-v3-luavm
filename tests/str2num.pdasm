PDVM 1.0
PLATFORM "pdcrt"

SECTION "code"
  LCONST 1
  MSG 0, 0, 1
  PRN
  NL

  LCONST 2
  MSG 0, 0, 1
  PRN
  NL

  LCONST 3
  MSG 0, 0, 1
  PRN
  NL

  LCONST 4
  MSG 0, 0, 1
  PRN
  NL

  LCONST 6
  MSG 5, 0, 1
  PRN
  NL

  LCONST 7
  MSG 5, 0, 1
  PRN
  NL

  LCONST 8
  MSG 5, 0, 1
  PRN
  NL

  LCONST 9
  MSG 5, 0, 1
  PRN
  NL
ENDSECTION

SECTION "procedures"
ENDSECTION

SECTION "constant pool"
  #0 STRING "comoNumeroEntero"
  #1 STRING "5837"
  #2 STRING "-731"
  #3 STRING "01"
  #4 STRING "13"
  #5 STRING "comoNumeroReal"
  #6 STRING "53.03"
  #7 STRING "-7.425"
  #8 STRING "00.42300000"
  #9 STRING "-0.13234"
ENDSECTION
