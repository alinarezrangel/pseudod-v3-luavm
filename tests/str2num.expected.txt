5837
-731
1
13
53.029999
-7.425000
0.423000
-0.132340
